ui/js/graphs.js

```diff
@@ -729,7 +745,7 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
                                             projects, optionCollectionMap,
                                             timeRange, thServiceDomain,
                                             PhSeries, defaultProjectName,
-                                            defaultPlatform, testsDisplayed) {
+                                            defaultPlatform, testsDisplayed, option, relateSeries) {
```

Comment:
Instead of defining "option" and "relateSeries" seperately, I'd just create a single object and pass it to this function, called "options".

Then you could just create the object as follows like so:

    { addRelatedPlatforms: true, relatedSeries: series }

If you were feeling ambitious, you could probably also merge defaultPlatform and testsDisplayed into this object.